server/routes/applicationsRoutes.ts

```diff
@@ -5,10 +5,12 @@ import {
   getStatuses,
   updateApplication,
   getApplicationById,
+  getAggregatedUserStats,
 } from "../controllers/applicationsController";
 
 const router = express.Router();
 
+router.get("/aggregated-user-stats/:userId", getAggregatedUserStats);
```

Comment:
Do we not want these routes to require auth? Can everyone view everyone else's application stats? People might at least want an option to make theirs private

Reply:
100% - Seeing as it is still int he dev environment I was going to push some refactor tickets for this. Let's discuss strategies for this ASAP. I refactored and added the protect middleware to all the routes. Code pushed.